utils/s2n_fork_detection.c

```diff
@@ -150,13 +151,16 @@ static void s2n_pthread_atfork_on_fork(void)
         abort();
     }
 }
+#endif
 
 static S2N_RESULT s2n_inititalise_pthread_atfork(void)
 {
     /* Register the fork handler pthread_atfork_on_fork that is excuted in the
      * child process after a fork.
      */
+#if !defined (__OpenBSD__)
```

Comment:
This needs a comment explaining the problem / why we can't use atfork in OpenBSD. I'm not a fan of only relying on git blame for documentation :)

Reply:
Fair enough :) I've added a comment. Please let me know if you feel it's sufficient.